packages/core/src/etherscan.ts

```diff
@@ -26,39 +26,22 @@ import {
 } from './utils'
 import { BuildInfo, SolcInput } from './languages'
 import {
+  fetchEtherscanConfigForNetwork,
   fetchNameForNetwork,
   isVerificationSupportedForNetwork,
 } from './networks'
 
 // Load environment variables from .env
 dotenv.config()
 
-/**
- * Get the current Etherscan chain config. This function copies some of the logic from Hardhat's
- * `getCurrentChainConfig`. We don't use their function because they use a different type of
- * provider (an `EthereumProvider`). ref:
- * https://github.com/NomicFoundation/hardhat/blob/2a99de5908cd56766c3a77e2088d6b9f82bd85ef/packages/hardhat-verify/src/internal/etherscan.ts#L46
- */
-export const getChainConfig = (chainId: number): ChainConfig => {
-  const chainConfig = [
-    // custom chains has higher precedence than builtin chains
-    ...[...customChains].reverse(), // the last entry has higher precedence
-  ].find((config) => config.chainId === chainId)
-
-  if (chainConfig === undefined) {
-    throw new Error(`Could not find chain config for: ${chainId}`)
-  }
-
-  return chainConfig
-}
-
```

Comment:
I removed this in favor of using a utility function that pulls the data from the `SPHINX_NETWORKS` array which is consistent with the way we handle all other network specific data.